llm-complete-guide/configs/production/embeddings.yaml

```
@@ -0,0 +1,48 @@
+# enable_cache: False
+
+# environment configuration
+settings:
+  docker:
+    parent_image: "zenmldocker/prepare-release:base-0.68.1"
+    requirements:
+      - langchain-community
+      - ratelimit
+      - langchain>=0.0.325
+      - langchain-openai
```

```suggestion
      - ratelimit
```

Not sure how these dependencies ended up here, but we don't need / want them.

---

might need to add openai